diagnostic.md

```diff
@@ -3,36 +3,48 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+faster loading speeds once page loads.
+routes make more sense.
+allows for quicker development--when understood.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+the handlebars templating library. It is handlebars.
```

Comment:
It has added functionality, such as the ability to call route or component templates, and some added helpers, such as `link-to`.
.5/1